filter.go

```diff
@@ -26,3 +26,65 @@ func extractMetrics(re *regexp.Regexp, in <-chan []byte, out chan<- []byte) {
 
 	close(out)
 }
+
+func SkipLuhnMatches(in <-chan []byte) <-chan []byte {
+	out := make(chan []byte)
+
+	re := regexp.MustCompile(`\D+`)
+	go skipLuhnMatches(re, in, out)
```

Comment:
It can't a constant but it could be a package level variable. I dislike those because it will increase the initialization time even when not used. I guess I don't mind too much so I can also do that if you insist.